great_expectations/checkpoint/checkpoint.py

```diff
@@ -400,3 +408,179 @@ def _get_batches_to_validate(self, batches):
                 batches_to_validate.append(batch)
 
         return batches_to_validate
+
+
+class ActionDicts:
+    STORE_VALIDATION_RESULT = {
+        "name": "store_validation_result",
+        "action": {"class_name": "StoreValidationResultAction"},
+    }
+    STORE_EVALUATION_PARAMS = {
+        "name": "store_evaluation_params",
+        "action": {"class_name": "StoreEvaluationParametersAction"},
+    }
+    UPDATE_DATA_DOCS = {
+        "name": "update_data_docs",
+        "action": {"class_name": "UpdateDataDocsAction", "site_names": None},
+    }
+
+    @staticmethod
+    def build_slack_action(webhook, notify_on, notify_with):
+        return {
+            "name": "slack_notification",
+            "action": {
+                "class_name": "SlackNotificationAction",
+                "slack_webhook": webhook,
+                "notify_on": notify_on,
+                "notify_with": notify_with,
+            },
+        }
+
+
+class SimpleCheckpoint(Checkpoint):
+    """
+    SimpleCheckpoint is an easy to use Checkpoint with enjoyable defaults.
+
+    This Checkopint performs the following actions by default:
+    1. store the validation result
+    2. store evaluation parameters
+    3. update all data docs sites
+
+    When configured, this Checkpoint can also send a slack message.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        data_context,
+        site_names: Optional[Union[str, List[str]]] = "all",
+        slack_webhook: Optional[str] = None,
+        notify_on: Optional[str] = "all",
+        notify_with: Optional[Union[str, List[str]]] = "all",
+    ):
+        # TODO nice docstring explaining arguments and example usages
+        self._site_names = site_names
+        self._validate_site_names()
```

Comment:
Love all the input validation going on - always so diligent @Aylr! Since we have the data_context, couldn't we make _validate_site_names smarter and check if any provided site_name strings are in the data_context config?

Reply:
Oh yes I love that idea. I will implement that.

Reply:
Done.